.gitignore

```diff
@@ -47,6 +47,9 @@ devopsLocal/
 pyclient/venv
 pyclient/dist
 pyclient/*.egg-info
+venv
+.venv
+*.egg-info
```

Comment:
Why were these removed?  I think the `venv` is there to handle venvs that pycharm creates, and the `egg-info` is for python files that are not checked in.

Reply:
These were added. And anything in `.gitignore` without a leading or middle slash will match at any level. So it is effectively the same as `**/venv/**` or "ignore any folder at any depth called `venv`". I could actually remove the `pyclient/venv` since it will match under this new item